src/renderer/components/ft-instance-selector/ft-instance-selector.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <ft-flex-box
+      class="settingsFlexStart460px"
+    >
+      <ft-input
+        :placeholder="placeholder"
+        :show-action-button="false"
+        :show-label="true"
+        :value="currentInstance"
+        :data-list="instanceList"
+        :tooltip="tooltip"
+        @input="handleInstanceInput"
+      />
+    </ft-flex-box>
+    <ft-flex-box>
+      <div v-if="backendType === 'piped'">
+        <a href="https://github.com/TeamPiped/Piped/wiki/Instances">
+          {{ $t('Settings.General Settings.View all Piped instance information') }}
+        </a>
+      </div>
+      <div
+        v-else
```

Comment:
```suggestion
        v-if="backendType === 'invidious'"
```
I believe this would be more clear written this way.